src/accelerator.rs

```diff
@@ -0,0 +1,98 @@
+use drivers::Driver;
+use Motion;
+use Operation;
+
+pub struct Accelerator<D: Driver> {
+    motions: Vec<Motion>,
+    driver: D,
+}
+
+impl <D: Driver> Accelerator<D> {
+    pub fn new(mots: Vec<Motion>, drv: D) -> Self {
+        Accelerator{motions: mots, driver: drv}
+    }
+    // Needs to be un pub.ed
```

Comment:
because it is only marked as pub for testing reasons, it shouldn't be. pub means that it is public.